src/auro-toaster.js

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2023 Alaska Airlines. All right reserved. Licensed under the Apache-2.0 license
+// See LICENSE in the project root for license information.
+
+// ---------------------------------------------------------------------
+
+// If using litElement base class
+import { LitElement } from "lit";
+import styleCss from "./style-toaster-css.js";
+
+export default class AuroToaster extends LitElement {
+  constructor() {
+    super();
+
+    /**
+     * @private
+     */
+    this.toasts = [];
+
+  }
+
+  connectedCallback() {
```

Comment:
Do we need to make a call to `disconnectedCallback` to clean up the event listeners?

Reply:
yes i believe we do! thanks for catching that. i pushed up some changes for this